events/README.md

```diff
@@ -0,0 +1,5 @@
+# Events
+
+This folder contains information and resources related to running events. Especially with regards to rules and the official code of conduct that everyone attending the Society's events must abide by.
+
+Everyone attending the Society's hackathons must abide by the [Hack Code of Conduct](https://hackcodeofconduct.org/).
```

Comment:
It's more than just _attendees_. As the site puts it: "attendees, sponsors, partners, volunteers and staff"